### DISCUSSION TITLE
Playing videos on web.telegram.org

### DISCUSSION THREAD

**Discussion by qupear** [qutebrowser/qutebrowser#7331] [Q&A] [answered]
Why videos and gif animations in Telegram web version may not load correctly?
All video messages have preview images and starts loading but this process cannot finish itself. In tg-K version videos seems to load but still cannot play.
Is it for everyone or just me?
All other functionality of this website seems to work fine.
![image](https://user-images.githubusercontent.com/8069685/182025451-279020ae-673f-4fc5-94b8-c018437f6beb.png)
**Added:** It's actually not only this site. reddit.com for example too don't play animations.

**Comment by toofar** [comment]
I don't have enough information to reproduce, so some questions:

1. can you check the console and network tab in `:devtools` for any errors?
2. can you provide an example reddit link that has this issue?

**Reply by qupear** [reply]
1. On console tab in :devtools I get these messages
```
Starting Raven release e3e285f5c81da8723aaf93eea34247f640d3a637-production public url https://6b5ce051b2ed4032bd906af3de0630cd@o418887.ingest.sentry.io/5667464
(index):1 A cookie associated with a cross-site resource at http://redditmedia.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
Failed to load resource: the server responded with a status of 451 () 472486.gif
472486.gif:1 GET https://id.rlcdn.com/472486.gif?gtmcb=651814927 451
Image (async)
kb @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:39
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:251
hc @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:52
e @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:163
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:31
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:164
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:164
$n @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:167
io @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:172
Ur @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:202
Xr @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:206
Zr @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:206
setTimeout (async)
L @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:39
as @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:209
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:262
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:259
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:259
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:266
```

And on network tab I have one line with red color which is 
```
Name=472486.gif?gtmcb=651814927, Status=451, Type=text/plain,Initiator=https://www.googletagmanager.com/gtm.js?id=GTM-5XVNS82&l=googleTagManager
curl "https://id.rlcdn.com/472486.gif?gtmcb=651814927" ^,Size=42B, Time=70ms

This is curl cmd:
  

> -H "authority: id.rlcdn.com" ^
>   -H "user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.15.2 Chrome/83.0.4103.122 Safari/537.36" ^
>   -H "accept: image/webp,image/apng,image/*,*/*;q=0.8" ^
>   -H "dnt: 1" ^
>   -H "accept-language: en-US,en;q=0.9" ^
>   -H "sec-fetch-site: cross-site" ^
>   -H "sec-fetch-mode: no-cors" ^
>   -H "sec-fetch-dest: image" ^
>   --compressed

```
2. https://www.reddit.com/r/gifs/comments/vvfuts/mobius_strip/
In other browsers this page displays an animation for me. Or at least I could open direct link https://preview.redd.it/ildc93m74na91.gif?format=mp4&s=48a695e4753ec7706daf6724bce339100d678e02 and see it. In qutebrowser this direct link in :devtools/console is empty and :devtools/Network Status=(canceled), Type=media, Initiator=Other, Size=0B. Format=mp4, Response=Failed to load response data

**Reply by toofar** [reply]
The preview on reddit for me is the one coming from https://preview.redd.it and it works fine for me in a temp basedir. Are you sure it's not the ad blocker in your main instance?

**Reply by qupear** [reply]
> The preview on reddit for me is the one coming from https://preview.redd.it and it works fine for me in a temp basedir. Are you sure it's not the ad blocker in your main instance?

I tried to delete pip adblock and this doesn't solve issue. Then tried this command (without success): `set -u *://preview.redd.it/* content.blocking.enabled false`

Is there any other setting I should try?

**Comment by The-Compiler** [comment]
So according to your output there:

>Failed to load resource: the server responded with a status of 451 () 472486.gif

You get a [HTTP 451](https://www.webfx.com/web-development/glossary/http-status-codes/what-is-a-451-status-code/), which is "Unavailable For Legal Reasons"... No idea why you would be getting that though!

Are you using some sort of proxy with qutebrowser but not other browsers, or the other way around? Can you reproduce when starting with `--temp-basedir`? Could you show your full `--version` information?

**Comment by qupear** [comment]
> Are you using some sort of proxy with qutebrowser but not other browsers, or the other way around?

I don't use any proxy. Just a direct connection same for all browsers.

> Can you reproduce when starting with --temp-basedir?

Yes, with --temp-basedir  video doesn't play. Here is a full Console output for same URL:

Starting Raven release efc437d89cfd8aebe0c46acce461dc2c5cf467a4-production public url https://6b5ce051b2ed4032bd906af3de0630cd@o418887.ingest.sentry.io/5667464
VM365:1 A cookie associated with a cross-site resource at http://redditmedia.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
id.rlcdn.com/472486.gif?gtmcb=2055199889:1 GET https://id.rlcdn.com/472486.gif?gtmcb=2055199889 net::ERR_NAME_NOT_RESOLVED
Image (async)
kb @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:39
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:251
hc @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:52
e @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:163
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:31
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:164
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:164
$n @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:167
io @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:172
Ur @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:202
Xr @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:206
Zr @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:206
setTimeout (async)
L @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:39
as @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:209
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:262
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:259
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:259
(anonymous) @ gtm.js?id=GTM-5XVNS82&l=googleTagManager:266
instrument.js:102 Not display reason: opt_out_or_no_session
VM457:1 A cookie associated with a cross-site resource at http://gfycat.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
app.669b9b3695b60769cc0b.js:2 GET https://metrics.gfycat.com/pix.gif?event=pageview_iframe&context=unrvd&app_id=com.gfycat.website&utc=8f5e54e5-6528-4b4d-9b58-9a2e8c8d1ee5&stc=b8908203-953b-43f5-bada-fcc5405933e7&ref=%2Fifr%2Fpowerlesspresentarcherfish&device_type=desktop&cache=1659417389902 net::ERR_NAME_NOT_RESOLVED
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
value @ app.669b9b3695b60769cc0b.js:2
fu @ app.669b9b3695b60769cc0b.js:2
Os @ app.669b9b3695b60769cc0b.js:2
t.unstable_runWithPriority @ app.669b9b3695b60769cc0b.js:2
Ho @ app.669b9b3695b60769cc0b.js:2
Ds @ app.669b9b3695b60769cc0b.js:2
ps @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.unstable_runWithPriority @ app.669b9b3695b60769cc0b.js:2
Ho @ app.669b9b3695b60769cc0b.js:2
zo @ app.669b9b3695b60769cc0b.js:2
Wo @ app.669b9b3695b60769cc0b.js:2
ss @ app.669b9b3695b60769cc0b.js:2
enqueueSetState @ app.669b9b3695b60769cc0b.js:2
m.setState @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
rs @ app.669b9b3695b60769cc0b.js:2
getComponent @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.mapAsync @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
l @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runEnterHooks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runChangeHooks @ app.669b9b3695b60769cc0b.js:2
m @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
c @ app.669b9b3695b60769cc0b.js:2
t.loopAsync @ app.669b9b3695b60769cc0b.js:2
p @ app.669b9b3695b60769cc0b.js:2
v @ app.669b9b3695b60769cc0b.js:2
r @ app.669b9b3695b60769cc0b.js:2
listen @ app.669b9b3695b60769cc0b.js:2
componentWillMount @ app.669b9b3695b60769cc0b.js:2
ya @ app.669b9b3695b60769cc0b.js:2
Gi @ app.669b9b3695b60769cc0b.js:2
ju @ app.669b9b3695b60769cc0b.js:2
Ss @ app.669b9b3695b60769cc0b.js:2
Ts @ app.669b9b3695b60769cc0b.js:2
_s @ app.669b9b3695b60769cc0b.js:2
ps @ app.669b9b3695b60769cc0b.js:2
ss @ app.669b9b3695b60769cc0b.js:2
Ks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
vs @ app.669b9b3695b60769cc0b.js:2
ec @ app.669b9b3695b60769cc0b.js:2
t.hydrate @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
rs @ app.669b9b3695b60769cc0b.js:2
getComponent @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.mapAsync @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
l @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runEnterHooks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runChangeHooks @ app.669b9b3695b60769cc0b.js:2
m @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
c @ app.669b9b3695b60769cc0b.js:2
t.loopAsync @ app.669b9b3695b60769cc0b.js:2
p @ app.669b9b3695b60769cc0b.js:2
v @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
Oh @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
39093 @ app.669b9b3695b60769cc0b.js:2
i @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Show 36 more frames
app.669b9b3695b60769cc0b.js:2 GET https://metrics.gfycat.com/pix.gif?event=pageview_iframe&context=unrvd&app_id=com.gfycat.website&utc=8f5e54e5-6528-4b4d-9b58-9a2e8c8d1ee5&stc=b8908203-953b-43f5-bada-fcc5405933e7&ref=%2Fifr%2Fsphericalfancycoelacanth&device_type=desktop&cache=1659417390001 net::ERR_NAME_NOT_RESOLVED
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
value @ app.669b9b3695b60769cc0b.js:2
fu @ app.669b9b3695b60769cc0b.js:2
Os @ app.669b9b3695b60769cc0b.js:2
t.unstable_runWithPriority @ app.669b9b3695b60769cc0b.js:2
Ho @ app.669b9b3695b60769cc0b.js:2
Ds @ app.669b9b3695b60769cc0b.js:2
ps @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.unstable_runWithPriority @ app.669b9b3695b60769cc0b.js:2
Ho @ app.669b9b3695b60769cc0b.js:2
zo @ app.669b9b3695b60769cc0b.js:2
Wo @ app.669b9b3695b60769cc0b.js:2
ss @ app.669b9b3695b60769cc0b.js:2
enqueueSetState @ app.669b9b3695b60769cc0b.js:2
m.setState @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
rs @ app.669b9b3695b60769cc0b.js:2
getComponent @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.mapAsync @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
l @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runEnterHooks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runChangeHooks @ app.669b9b3695b60769cc0b.js:2
m @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
c @ app.669b9b3695b60769cc0b.js:2
t.loopAsync @ app.669b9b3695b60769cc0b.js:2
p @ app.669b9b3695b60769cc0b.js:2
v @ app.669b9b3695b60769cc0b.js:2
r @ app.669b9b3695b60769cc0b.js:2
listen @ app.669b9b3695b60769cc0b.js:2
componentWillMount @ app.669b9b3695b60769cc0b.js:2
ya @ app.669b9b3695b60769cc0b.js:2
Gi @ app.669b9b3695b60769cc0b.js:2
ju @ app.669b9b3695b60769cc0b.js:2
Ss @ app.669b9b3695b60769cc0b.js:2
Ts @ app.669b9b3695b60769cc0b.js:2
_s @ app.669b9b3695b60769cc0b.js:2
ps @ app.669b9b3695b60769cc0b.js:2
ss @ app.669b9b3695b60769cc0b.js:2
Ks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
vs @ app.669b9b3695b60769cc0b.js:2
ec @ app.669b9b3695b60769cc0b.js:2
t.hydrate @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
rs @ app.669b9b3695b60769cc0b.js:2
getComponent @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.mapAsync @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
l @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runEnterHooks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runChangeHooks @ app.669b9b3695b60769cc0b.js:2
m @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
c @ app.669b9b3695b60769cc0b.js:2
t.loopAsync @ app.669b9b3695b60769cc0b.js:2
p @ app.669b9b3695b60769cc0b.js:2
v @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
Oh @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
39093 @ app.669b9b3695b60769cc0b.js:2
i @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Show 36 more frames
powerlesspresentarcherfish:1 Uncaught (in promise) TypeError: Failed to fetch
app.669b9b3695b60769cc0b.js:2 GET https://metrics.gfycat.com/pix.gif?event=pageview_iframe&context=unrvd&app_id=com.gfycat.website&utc=8f5e54e5-6528-4b4d-9b58-9a2e8c8d1ee5&stc=b8908203-953b-43f5-bada-fcc5405933e7&ref=%2Fifr%2Fgiganticcompletealbino&device_type=desktop&cache=1659417390091 net::ERR_NAME_NOT_RESOLVED
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
value @ app.669b9b3695b60769cc0b.js:2
fu @ app.669b9b3695b60769cc0b.js:2
Os @ app.669b9b3695b60769cc0b.js:2
t.unstable_runWithPriority @ app.669b9b3695b60769cc0b.js:2
Ho @ app.669b9b3695b60769cc0b.js:2
Ds @ app.669b9b3695b60769cc0b.js:2
ps @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.unstable_runWithPriority @ app.669b9b3695b60769cc0b.js:2
Ho @ app.669b9b3695b60769cc0b.js:2
zo @ app.669b9b3695b60769cc0b.js:2
Wo @ app.669b9b3695b60769cc0b.js:2
ss @ app.669b9b3695b60769cc0b.js:2
enqueueSetState @ app.669b9b3695b60769cc0b.js:2
m.setState @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
rs @ app.669b9b3695b60769cc0b.js:2
getComponent @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.mapAsync @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
l @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runEnterHooks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runChangeHooks @ app.669b9b3695b60769cc0b.js:2
m @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
c @ app.669b9b3695b60769cc0b.js:2
t.loopAsync @ app.669b9b3695b60769cc0b.js:2
p @ app.669b9b3695b60769cc0b.js:2
v @ app.669b9b3695b60769cc0b.js:2
r @ app.669b9b3695b60769cc0b.js:2
listen @ app.669b9b3695b60769cc0b.js:2
componentWillMount @ app.669b9b3695b60769cc0b.js:2
ya @ app.669b9b3695b60769cc0b.js:2
Gi @ app.669b9b3695b60769cc0b.js:2
ju @ app.669b9b3695b60769cc0b.js:2
Ss @ app.669b9b3695b60769cc0b.js:2
Ts @ app.669b9b3695b60769cc0b.js:2
_s @ app.669b9b3695b60769cc0b.js:2
ps @ app.669b9b3695b60769cc0b.js:2
ss @ app.669b9b3695b60769cc0b.js:2
Ks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
vs @ app.669b9b3695b60769cc0b.js:2
ec @ app.669b9b3695b60769cc0b.js:2
t.hydrate @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
rs @ app.669b9b3695b60769cc0b.js:2
getComponent @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
t.mapAsync @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
l @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runEnterHooks @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
a @ app.669b9b3695b60769cc0b.js:2
runChangeHooks @ app.669b9b3695b60769cc0b.js:2
m @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
c @ app.669b9b3695b60769cc0b.js:2
t.loopAsync @ app.669b9b3695b60769cc0b.js:2
p @ app.669b9b3695b60769cc0b.js:2
v @ app.669b9b3695b60769cc0b.js:2
t.default @ app.669b9b3695b60769cc0b.js:2
Oh @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Promise.then (async)
39093 @ app.669b9b3695b60769cc0b.js:2
i @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
(anonymous) @ app.669b9b3695b60769cc0b.js:2
Show 36 more frames
sphericalfancycoelacanth:1 Uncaught (in promise) TypeError: Failed to fetch
giganticcompletealbino:1 Uncaught (in promise) TypeError: Failed to fetch

On Network tab there are now 5 red lines
1) https://id.rlcdn.com/472486.gif?gtmcb=2055199889
(failed) net::ERR_NAME_NOT_RESOLVED
2) https://metrics.gfycat.com/pix.gif?event=pageview_iframe&context=unrvd&app_id=com.gfycat.website&utc=8f5e54e5-6528-4b4d-9b58-9a2e8c8d1ee5&stc=b8908203-953b-43f5-bada-fcc5405933e7&ref=%2Fifr%2Fpowerlesspresentarcherfish&device_type=desktop&cache=1659417389902
(failed), fetch
3) https://metrics.gfycat.com/pix.gif?event=pageview_iframe&context=unrvd&app_id=com.gfycat.website&utc=8f5e54e5-6528-4b4d-9b58-9a2e8c8d1ee5&stc=b8908203-953b-43f5-bada-fcc5405933e7&ref=%2Fifr%2Fsphericalfancycoelacanth&device_type=desktop&cache=1659417390001
(failed), fetch
4) https://metrics.gfycat.com/pix.gif?event=pageview_iframe&context=unrvd&app_id=com.gfycat.website&utc=8f5e54e5-6528-4b4d-9b58-9a2e8c8d1ee5&stc=b8908203-953b-43f5-bada-fcc5405933e7&ref=%2Fifr%2Fgiganticcompletealbino&device_type=desktop&cache=1659417390091
(failed), fetch
5) https://thumbs.gfycat.com/SphericalFancyCoelacanth-mobile.mp4
(canceled), media

> Could you show your full --version information?

qutebrowser v2.5.2
Git commit: f911ffd55 on v2.5.x (2022-06-22 16:04:52 +0200)
Backend: QtWebEngine 5.15.2, based on Chromium 83.0.4103.122
Qt: 5.15.2

CPython: 3.9.13
PyQt: 5.15.6

sip: 6.5.1
colorama: 0.4.4
jinja2: 3.1.1
pygments: 2.11.2
yaml: 6.0
adblock: 0.5.2
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebEngine: 5.15.5
PyQt5.QtWebKitWidgets: no
pdf.js: 2.14.305 (bundled)
sqlite: 3.33.0
QtNetwork SSL: OpenSSL 1.1.1b  26 Feb 2019

Style: QWindowsVistaStyle
Platform plugin: windows
OpenGL: Intel, 4.6.0 - Build 30.0.101.1338
Platform: Windows-10-10.0.19043-SP0, 64bit
Frozen: True
Imported from C:\qutebrowser\qutebrowser
Using Python from C:\qutebrowser\qutebrowser.exe
Qt library executable path: C:/qutebrowser/PyQt5/Qt5/bin, data path: C:/qutebrowser/PyQt5/Qt5
OS Version: 10, 10.0.19043, SP0, Multiprocessor Free

Paths:
cache: C:\Users\User\AppData\Local\Temp\qutebrowser-basedir-iz9kcku5\cache
config: C:\Users\User\AppData\Local\Temp\qutebrowser-basedir-iz9kcku5\config
data: C:\Users\User\AppData\Local\Temp\qutebrowser-basedir-iz9kcku5\data
runtime: C:\Users\User\AppData\Local\Temp\qutebrowser

Autoconfig loaded: yes
Config.py: no config.py was loaded
Uptime: 0:00:02

**Comment by The-Compiler** [comment, marked as answer]
Sorry for the late answer, this got lost a bit with my holidays. The HTTP 451 was actually a red herring there - according to your version output, you're using the Windows build. That (or rather the underlying QtWebEngine binary release) does not support proprietary codecs, which is why those videos don't play.

**Reply by qupear** [reply]
I get it, thanks for clarification. Any chance proprietary codecs will become supported in the future releases?
Will try a Linux build in WSL, maybe that will work for me.
